tests/aggregation/tests.py

```diff
@@ -1315,3 +1315,34 @@ def test_aggregation_subquery_annotation_related_field(self):
         # with self.assertNumQueries(1) as ctx:
         #     self.assertSequenceEqual(books_qs, [book])
         # self.assertEqual(ctx[0]['sql'].count('SELECT'), 2)
+
+    def test_ticket26390(self):
```

Comment:
Don't include tickets numbers in test methods, e.g.

```suggestion
    def test_aggregation_random_ordering(self):
```